lib/trino-parquet/src/main/java/io/trino/parquet/writer/PrimitiveColumnWriter.java

```diff
@@ -321,11 +324,6 @@ public long getRetainedBytes()
 
     private void updateBufferedBytes()
     {
-        // Avoid using streams here for performance reasons
-        long pageBufferedBytes = 0;
-        for (ParquetDataOutput output : pageBuffer) {
-            pageBufferedBytes += output.size();
-        }
         bufferedBytes = pageBufferedBytes +
```

Comment:
why can't we do same for `bufferedBytes`?

Reply:
It's a lot more complicated to track deltas to ValuesWriter sizes and getting those directly from the ValuesWriter is relatively cheap.